std_msgs/Header header
sensor_msgs/Image depthmap       
std_msgs/Float32MultiArray xu_lookup
std_msgs/Float32MultiArray yv_lookup